### DISCUSSION TITLE
Structure of HIG Document

### DISCUSSION THREAD

**Discussion by urishiraval** [HEmile/obsidian-api-docs#5] [Ideas]
From [wiki](https://en.wikipedia.org/wiki/Human_interface_guidelines):

"HIGs are recommendations and advice meant to help developers create better applications."

So I think we should (as headings) have:

# Plugin Structure

> Maybe have a case study type thing breaking down a community plugin that's well structured 

# Interface Guidelines

### Registering Shortcuts

> etc.....

# Data Source Guidelines

### Using the Adapter Design Pattern

> etc...

# Community Plugins

> List of plugins

---

At least those are the things on the top of my head

**Comment by HEmile** [comment]
Would this be a single document with a set of headers? If we're doing an Obsidian vault, we might as well split these up into different notes, right :)?
